Meridian59.DebugUI/ObjectEditor.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.ComponentModel;
+using System.Windows.Forms;
+using Meridian59.Data.Models.AdminData;
+
+namespace Meridian59.DebugUI
+{
+    public partial class ObjectEditor : Form
```

Comment:
Usually the classname of all views refers to the model name that is shown in the view, e.g.:
* Models.RoomInfo --> RoomInfoView
* Models.ObjectBase --> ObjectBaseView

It looks like a proper name would be "AdminObjectView", because this class seems to display the data of the "AdminObject" model.